.github/workflows/deploy_to_s3_prod.yml

```diff
@@ -15,49 +15,50 @@ jobs:
       url: "https://auth.avantifellows.org"
 
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-
-    - name: Cache node modules
-      uses: actions/cache@v1
-      with:
-        path: node_modules
-        key: ${{ runner.OS }}-build-${{ hashFiles('**/package-lock.json') }}
-        restore-keys: |
-          ${{ runner.OS }}-build-
-          ${{ runner.OS }}-
-
-    - name: Install
-      run: npm install
-
-    # build the vuepress docs
-    - name: Build
-      env:
-        NODE_ENV: production
-        VITE_APP_PLIO_AF_API_KEY: ${{ secrets.VITE_APP_PLIO_AF_API_KEY }}
-        VITE_APP_BASE_URL_PLIO: 'https://app.plio.in/#/af/play/'
-        VITE_APP_QUIZ_AF_API_KEY: ${{ secrets.VITE_APP_QUIZ_AF_API_KEY }}
-        VITE_APP_BASE_URL_QUIZ: 'https://quiz.avantifellows.org/quiz/'
-        VITE_APP_AWS_ACCESS_KEY_ID: ${{ secrets.VITE_APP_AWS_SQS_ACCESS_KEY }}
-        VITE_APP_AWS_SECRET_KEY: ${{ secrets.VITE_APP_AWS_SQS_SECRET_ACCESS_KEY }}
-        VITE_APP_BASE_URL_MEET: 'https://meet.google.com/'
-        VITE_APP_OTP_SERVICE: ${{ secrets.VITE_APP_OTP_SERVICE }}
-        VITE_APP_AWS_REGION: ${{ secrets.VITE_APP_AWS_REGION }}
-        VITE_APP_AWS_SQS_URL: ${{ secrets.VITE_APP_AWS_SQS_URL }}
-        VITE_APP_BASE_URL_YOUTUBE: 'https://www.youtube.com/watch?v='
-        VITE_APP_CHECK_USER_ENDPOINT: '/checkForUser'
-        VITE_APP_GET_GROUP_ENDPOINT: '/getGroupData'
-        VITE_APP_CLOUD_FUNCTIONS_BASE_URL: "https://asia-south1-avantifellows.cloudfunctions.net"
-
-      run: npm run build
-
-    - name: Configure AWS Credentials
-      uses: aws-actions/configure-aws-credentials@v1
-      with:
-        aws-access-key-id: ${{ secrets.AWS_S3_ACCESS_KEY_ID }}
-        aws-secret-access-key: ${{ secrets.AWS_S3_SECRET_ACCESS_KEY }}
-        aws-region: ap-south-1
-
-    # sync with the given S3 bucket
-    - name: Deploy static site to S3 bucket
-      run: aws s3 sync ./dist s3://af-auth-layer --delete --acl public-read
+      - name: Checkout
```

Comment:
Why did you change the indentation? Doesn't seem to be changed for other sections